utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -35,21 +35,36 @@ THE SOFTWARE.
 #include <omp.h>
 #include <fstream>
 #include <turbojpeg.h>
+#include <boost/filesystem.hpp>
+
+#ifdef __HIP_PLATFORM_HCC__
+#include <hip/hip_fp16.h>
```

Comment:
Indent the code in this if  else condition
it is not aligned properly

Reply:
Done